helm-chart/renku-notebooks/values.yaml

```diff
@@ -44,7 +44,7 @@ autoscaling:
 
 image:
   repository: renku/renku-notebooks
-  tag: 'latest'
+  tag: '1.0.0'
```

Comment:
Can you run `pipenv run chartpress --reset` on the chart? I think it's nice to have latest in here and have chartpress replace it before publishing whenever we tag a release.